applications/app/views/videoEmbed.scala.html

```diff
@@ -78,7 +78,7 @@
                                 || 'XDomainRequest' in window
                             )
                         ),
-                        config: @fragments.javaScriptConfig(model.SimpleContentPage(video))
+                        config: @Html(templates.js.javaScriptConfig(model.SimpleContentPage(video)).body)
```

Comment:
Ah okay, because it's coming through the HTML template parsing, you need to turn the encoding off... :+1: